src/Components/Table/DjTable.tsx

```diff
@@ -85,46 +92,52 @@ export default class DjTable extends React.Component<
     this.redirect = this.redirect.bind(this)
   }
 
-  handleChange(pagination: any, filters: Record<string, FilterValue | null>, sorter: any) {
-
-    let filter = []
-
+  handleChange(
+    pagination: any,
+    filters: Record<string, FilterValue | null>,
+    sorter: any
+  ) {
+    let filter: { [key: string]: djFilter } = {}
     let isFilterNull = true
 
-    for(const key in filters){
-        if(filters[key] !== null){
-          filter.push(`&${key}=${filters[key]}`)
-          isFilterNull = false
+    for (const key in filters) {
+      if (filters[key] !== null) {
+        filter[`${key}`] = {
+          filteredValue: filters[key],
+          restriction: `&${key}=${filters[key]}`,
+          filtered: true,
         }
+        isFilterNull = false
+      }
     }
 
     let sorterArr = []
 
     let isSorterNull = true
 
-    if(sorter["order"] !== null && sorter["field"] !== null){
+    if (sorter['order'] !== null && sorter['field'] !== null) {
```

Comment:
I suspect this auto-linting issue came up since you and @zitrosolrac aren't aligned on the same linter? :shrug:

Maybe double check to prevent it from cluttering the diff.